app/controllers/partners/families_controller.rb

```diff
@@ -9,7 +9,8 @@ def index
       @filterrific = initialize_filterrific(
         current_partner.families
                        .order(sort_order),
-        params[:filterrific]
+        params[:filterrific],
+        default_filter_params: {"include_archived" => 0}
```

Comment:
Why is this zero rather than true/false?

Reply:
It is zero because that is what is expected in the scope because that is what is returned in the params on the controller

Reply:
I'd rather have something on the controller side managing the translation from 0 -> false, 1 -> true so that all other code can deal with true/false rather than tying e.g. model code to the parameters coming from a controller.